package.json

```diff
@@ -21,20 +21,20 @@
   },
   "homepage": "https://github.com/PaystackHQ/nigerialogos#readme",
   "devDependencies": {
-    "node-sass": "^4.11.0",
     "eslint": "^6.1.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.3.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.1.0",
     "husky": "^3.0.5",
     "lint-staged": "^9.2.5",
+    "node-sass": "^4.13.1",
```

Comment:
Hey @ichtrojan just curious, is there any specific reason why you updated the node-sass version? Did you run into any form of error with the previous version?

Reply:
@ichtrojan

Reply:
Yes, it failed to build on macOS Catalina.

Thanks for the merge.